File: Python/Cpu.py
from Komponen import Komponen

class Cpu(Komponen):
    def __init__(self, nama, core, kecepatan):
        super().__init__(nama)
        self.core = core
        self.kecepatan = kecepatan
    
    def get_info(self):
        return f"{self.nama} ({self.core} Core) ~{self.kecepatan}GHz"

File: CPP/Cpu.cpp
#include "Komponen.cpp"

class Cpu : public Komponen {
private:
    int core;
    float kecepatan;
public:
    Cpu() {}
    Cpu(string nama, int core, float kecepatan) : Komponen(nama) {
        this->core = core;
        this->kecepatan = kecepatan;
    }
    void tampil() {
        cout << "Cpu       : " << nama << " (" << core << " Core) ~" << kecepatan << "GHz" << endl;
    }
};